Fastfile

```diff
@@ -74,6 +92,7 @@ default_platform :ios
   end
 
   after_all do |lane|
+    notifiy_slack()
```

Comment:
@KaneCheshire yeah this is a limitation, the advantage of having this in an after_all is no matter what lane gets called this is guaranteed. The downside is there is no calling super as it where in fastlane. The user can call `notify_slack` themselves, do people think its worth losing this happening explicitly after every lane for the case where someone wants there own `after_all`? if we introduce multiple lanes that need it we could risk slack being called multiple times when people mix and match lanes which could get messy